src/alpine/manifest.json

```diff
@@ -232,6 +232,22 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/alpine/3.17/amd64",
+              "os": "linux",
+              "osVersion": "alpine3.17",
+              "tags": {
+                "alpine-3.17-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "alpine-3.17$(FloatingTagSuffix)": {},
+                "alpine-3.17-local": {
```

Comment:
This tag is not needed since there are no other Dockerfile dependencies on this image.

Reply:
Removed. Thanks!